### PR TITLE
Change final form page if jurisdiction is full

### PR DESCRIPTION
**Pull request by Samcbehrens**
Change information presented if the jurisdiction is Full
Screen Shot 2020-10-13 at 9 43 19 PM

### CODE REVIEW

**Review by mojowen — DISMISSED** [review]

File: Readme.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ This is currently done manually
 #### Developing the site and form in tandem
 
 1. In `/site`, remove the npm version and reference the local version with `npm uninstall @ptp-us/power-the-polls-form` and then `npm install ../power-the-polls-form`
-2. In `page-forn.tsx` change the import from `import "@ptp-us/power-the-polls-form";` to `import "@ptp-us/power-the-polls-form/dist";`
-3. Use `npm run dev:form` which will build the form and then `npm run dev`
+2. In `page-form.tsx` change the import from `import "@ptp-us/power-the-polls-form";` to `import "@ptp-us/power-the-polls-form/dist";`
+3. In `/power-the-polls-form` Use `npm run build` which will build the form and then  in `/site` run `npm run dev`
```
Comment (nexussays) [inline Readme.md:54]: Thank you!

File: power-the-polls-form/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
       "dev:site": "npm run clean && stencil build --watch",
       "build": "npm run lint && stencil build --prod --docs --no-cache && terser ./dist/full/index.js -o ./dist/full/index.js",
       "build:clean": "npm run clean-all && npm run build",
-      "package": "copyfiles -E -u 1 ./dist/**/* ../dist/power-the-polls-form/ && copyfiles -E package.json Readme.md LICENSE ../dist/power-the-polls-form/ && replace \"dist/\" \"\" ../dist/power-the-polls-form/package.json && cross-env-shell replace \"CURRENT_VERSION\" \"$npm_package_version\" ../dist/power-the-polls-form/  -r --include=\"*.js\"",
+      "package": "copyfiles -E -u 1 \"./dist/**/*\" \"../dist/power-the-polls-form/\" && copyfiles -E package.json Readme.md LICENSE ../dist/power-the-polls-form/ && replace \"dist/\" \"\" ../dist/power-the-polls-form/package.json && cross-env-shell replace \"CURRENT_VERSION\" \"$npm_package_version\" ../dist/power-the-polls-form/  -r --include=\"*.js\"",
```
Comment (nexussays) [inline power-the-polls-form/package.json:24]: -.-

File: power-the-polls-form/tsconfig.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
    },
    "include": [
       "src",
-      "types/jsx.d.ts"
+      "types/jsx.d.ts", "../scripts/github-actions/getFullJurisdictions.ts"
```
Comment (nexussays) [inline power-the-polls-form/tsconfig.json]: VSCode auto-add?

**Review by nexussays — DISMISSED** [review]

File: power-the-polls-form/tsconfig.json (hunk 3 above)
Comment (Samcbehrens) [reply power-the-polls-form/tsconfig.json]: yep, it is from a branch that I pull off from this work. Good catch.

**Review by nexussays — DISMISSED** [review]

**Review by nexussays — APPROVED** [review]

**Pull request merged**